.eslintrc.json

```diff
@@ -43,7 +43,7 @@
         "@next/next/no-img-element": "off", // We currently not using next/image because it isn't supported with SSG mode
         "@typescript-eslint/comma-dangle": "off",
         "no-underscore-dangle": "off",
-        "@typescript-eslint/consistent-type-imports": "error", // Ensure `import type` is used when it's necessary
+        "@typescript-eslint/consistent-type-imports": "off", // Ensure `import type` is used when it's necessary
```

Comment:
[Question] With the new auto-fix, does this still cause issues?

Reply:
ah it goes back to the undefined issue in notion.ts if it put it back to error :/